GIBSDownloader/gibs_downloader.py

```diff
@@ -111,6 +112,8 @@ def main():
     parser.add_argument("--product", default=Product.viirs, type=Product, help="select the NASA imagery product", choices=list(Product))
```

Comment:
default value for product None. Change name argument default to be product.viiirs,.25.

Implement a check that if product is not None, change name to product.get_long_name(),.25